source/help/settings/theme-colors.rst

```diff
@@ -160,3 +160,13 @@ Discord Dark Theme
 .. code-block:: none
 
   {"sidebarBg":"#2f3136","sidebarText":"#ffffff","sidebarUnreadText":"#ffffff","sidebarTextHoverBg":"#33363c","sidebarTextActiveBorder":"#66cfa0","sidebarTextActiveColor":"#ffffff","sidebarHeaderBg":"#27292c","sidebarHeaderTextColor":"#ffffff","onlineIndicator":"#43b581","awayIndicator":"#faa61a","dndIndicator":"#f04747","mentionBg":"#6e84d2","mentionBg":"#6e84d2","mentionColor":"#ffffff","centerChannelBg":"#36393f","centerChannelColor":"#dddddd","newMessageSeparator":"#6e84d2","linkColor":"#2095e8","buttonBg":"#43b581","buttonColor":"#ffffff","errorTextColor":"#ff6461","mentionHighlightBg":"#3d414f","mentionHighlightLink":"#6e84d2","codeTheme":"monokai"}
+
+Night Owl Dark theme
+^^^^^^^^^^^^^^^^^^^^
+
+.. image:: ../../images/nightOwlDark.PNG
+  :alt: nightOwlDark
+
+.. code-block:: none
+
+  {"sidebarBg":"#011627","sidebarText":"#d6deeb","sidebarUnreadText":"#d6deeb","sidebarTextHoverBg":"#1d3b53","sidebarTextActiveBorder":"#ff2c83","sidebarTextActiveColor":"#82aaff","sidebarHeaderBg":"#1d3b53","sidebarHeaderTextColor":"#d6deeb","onlineIndicator":"#addb67","awayIndicator":"#ffbc42","dndIndicator":"#f74343","mentionBg":"#7e57c2","mentionBj":"#011627","mentionColor":"#145dbf","centerChannelBg":"#011627","centerChannelColor":"#d6deeb","newMessageSeparator":"#ff8800","linkColor":"#2389d7","buttonBg":"#166de0","buttonColor":"#011627","errorTextColor":"#fd5960","mentionHighlightBg":"#ffe577","mentionHighlightLink":"#166de0","codeTheme":"solarized-dark"}
```

Comment:
@CrHasher please update mentionBj > mentionBg

cc// @justinegeffen

Reply:
@CrHasher what server version were these values generated on? I'm seeing bugs in the theme when I try this myself

![image](https://user-images.githubusercontent.com/14333569/101681824-9bf33c00-3a17-11eb-9e51-525d8666c99b.png)

Can you try on https://community.mattermost.com/ so we are sure the theme works on our newest versions?